nixos/modules/services/misc/paperless.nix

```diff
@@ -82,7 +82,7 @@ let
   };
 in
 {
-  meta.maintainers = with lib.maintainers; [ leona SuperSandro2000 erikarvstedt ];
+  meta.maintainers = with lib.maintainers; [ leona SuperSandro2000 erikarvstedt atemu theuni ];
```

Comment:
So many maintainers will probably just diverge responsibility and in the end no one feels really responsible.